pex/pex.py

```diff
@@ -155,9 +155,14 @@ def minimum_sys_modules(cls, site_libs, modules=None):
 
   @classmethod
   def minimum_sys_path(cls, site_libs, inherit_path):
+    ver = sys.version[:3]
     scrub_paths = OrderedSet()
     site_distributions = OrderedSet()
     user_site_distributions = OrderedSet()
+    undesirable_paths = OrderedSet([
+      '/System/Library/Frameworks/Python.framework/Versions/%s/Extras/lib/python' % ver,
```

Comment:
yeh, the Extras.pth in site-packages seems to be the primary linkage here.

I alluded to this in #312, but I think a semi-sane workaround approach here to identifying these paths (which could merge directly into `_extras_paths`) could be:

```
>>> def yield_extras_paths():
...   extras_prefix = os.path.join(sys.prefix, 'Extras')
...   for path in sys.path:
...     if path.startswith(extras_prefix):
...       yield path
... 
>>> extras_paths = list(yield_extras_paths())
>>> pprint(extras_paths)
['/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python',
 '/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/PyObjC']
```